frontend/language/src/nb.json

```
@@ -1838,6 +1839,8 @@
   "ux_editor.properties_panel.subform_table_columns.choose_component_description": "Listen viser bare de komponentene som har minst én datamodellbinding og en ledetekst",
   "ux_editor.properties_panel.subform_table_columns.column_cell_content": "Celleinnhold (query):",
   "ux_editor.properties_panel.subform_table_columns.column_header": "Kolonne {{columnNumber}}",
+  "ux_editor.properties_panel.subform_table_columns.column_multiple_data_model_bindings_description": "Komponenten inneholder flere felt og kan kun vise et felt per kolonne.",
```

I'd suggest removing the `.` at the end of the sentence to match with `x_editor.properties_panel.subform_table_columns.choose_component_description`

```suggestion
  "ux_editor.properties_panel.subform_table_columns.column_multiple_data_model_bindings_description": "Komponenten inneholder flere felt og kan kun vise et felt per kolonne",
```

---

FIxed 👍